cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -352,7 +353,7 @@ function Multiselect( element ) { // eslint-disable-line max-statements, inline-
   function _bindEvents() {
     var inputs = _list.querySelectorAll( 'input' );
 
-    _bind( _search, {
+    domEvents.bindEvent( _search, {
```

Comment:
It seems like it might be easier to set the events on `_search`'s parent and then in the event handlers get `event.target`. See http://codepen.io/anon/pen/aNWXYZ
Maybe out of scope with this PR, but a TODO would be good if that makes sense.